src/main.rs

```diff
@@ -24,25 +31,34 @@ fn time_to_mine(block_height: u64) -> Duration {
     // when the lazy macro is expanded
     // if a value has a static lifetime then it means that value lives as long as the program lives
     let rpc_client: &Client = &*RPC_CLIENT;
-    rpc_client.get_block_hash(234);
-    todo!()
+    let current_block: Block = get_block(block_height, rpc_client).unwrap();
+    let next_block: Block = get_block(block_height + 1, rpc_client).unwrap();
```

Comment:
I think what you're looking for is the previous block and not the next one.

What would happen if you enter a block height that is at the top of the chain? The next block wouldn't exist yet and you wouldn't be able to get a timestamp.

Reply:
i have fixed this